bin/plugin/commands/test-plugins.js

```diff
@@ -0,0 +1,323 @@
+				)
+			);
+		} catch ( e ) {
+			log(
+				formats.error(
+					`Error copying test assets for plugin "${ plugin }". ${ e }`
+				)
+			);
+
+			// Return with exit code 1 to trigger a failure in the test pipeline.
+			process.exit( 1 );
+		}
+
+		// Execute composer install within built plugin following copy.
+		execSync(
+			`composer install --working-dir=${ builtPluginsDir }${ plugin } --no-interaction`,
+			( err, output ) => {
+				// once the command has completed, the callback function is called
+				if ( err ) {
+					log( formats.error( `${ err }` ) );
+					return;
+				}
+				// log the output received from the command
+				log( output );
+			}
+		);
+	} );
+
+	// Amend wp-env.json to reference built plugins only.
+	log(
+		formats.success(
+			`\nTest assets copied and "composer install" executed for all built plugins, configuring .wp-env.json file to reference built plugins as active plugins. Plugins to be enabled are: ${ builtPlugins.join(
+				', '
+			) }`
+		)
+	);
+
+	// Buffer .wp-env.json content var.
+	let wpEnvFileContent = '';
+
+	try {
+		wpEnvFileContent = fs.readFileSync( wpEnvFile, 'utf-8' );
+	} catch ( e ) {
+		log( formats.error( `Error reading file "${ wpEnvFile }": "${ e }"` ) );
+
+		// Return with exit code 1 to trigger a failure in the test pipeline.
+		process.exit( 1 );
+	}
+
+	// If the contents of the file were incorrectly read or exception was not captured and value is blank, abort.
+	if ( '' === wpEnvFileContent ) {
+		log(
+			formats.error(
+				`File content for "${ wpEnvFile }" is empty, aborting.`
+			)
+		);
+
+		// Return with exit code 1 to trigger a failure in the test pipeline.
+		process.exit( 1 );
+	}
+
+	// If we do not have a match on the wp-env enabled plugins regex, abort.
+	if ( ! wpEnvPluginsRegex.test( wpEnvFileContent ) ) {
+		log(
+			formats.error(
+				`Unable to find plugins property/key in WP Env config file: "${ wpEnvFile }". Please ensure that it is present and try agagin.`
+			)
+		);
+
+		// Return with exit code 1 to trigger a failure in the test pipeline.
+		process.exit( 1 );
+	}
+
+	// Let the user know we're re-writing the .wp-env.json file.
+	log( formats.success( `Rewriting plugins property in ${ wpEnvFile }` ) );
+
+	// Attempt replacement of the plugins property in .wp-env.json file to match built plugins.
+	try {
+		// Create plugins property from built plugins.
+		wpEnvPluginsRegexReplacement = `"plugins": [ "${ builtPlugins
+			.map( ( item ) => `${ builtPluginsDir }${ item }` )
+			.join( '", "' ) }" ],`;
+
+		fs.writeFileSync(
+			wpEnvFile,
+			wpEnvFileContent.replace(
+				wpEnvPluginsRegex,
+				wpEnvPluginsRegexReplacement
+			)
+		);
+	} catch ( e ) {
+		log(
+			formats.error(
+				`Error replacing content in ${ wpEnvFile } using regex "${ wpEnvPluginsRegex }": "${ e }"`
+			)
+		);
+
+		// Return with exit code 1 to trigger a failure in the test pipeline.
+		process.exit( 1 );
+	}
```

Comment:
Maybe move this code into its own function? The handler function is extremely long, and I think especially this here is pretty much one logical step that could be broken apart into its own function unit.